src/App.js

```diff
@@ -4,35 +4,29 @@ import { Route, Routes } from "react-router-dom";
 import ComponentsTesting from "./pages/componentsTesting/ComponentsTesting";
 import Homepage from "./pages/Homepage";
 import NewPost from "./pages/newPost/NewPost.jsx";
-import NewCommunity from "./pages/newCommunity/NewCommunity.jsx";
 import ForgotPasswordPage from "./pages/LoginRegister/ForgotPasswordPage";
 import LoginPage from "./pages/LoginRegister/LoginPage";
 import RegisterPage from "./pages/LoginRegister/RegisterPage";
 import NavBar from "./components/NavBar/NavBar";
-import CommunitiesPage from "./pages/Communities/CommunitiesPage";
 import Posts from "./pages/Posts/Posts";
+import Community from './pages/Community'
 
```

Comment:
```suggestion
import CommunitiesPage from "./pages/Communities/CommunitiesPage";
import NewCommunity from "./pages/newCommunity/NewCommunity.jsx";
```

Reply:
i think his community is my communitiesPage, just need to replace it